tests/core/views/test_api.py

```diff
@@ -153,7 +153,11 @@ async def test_pay_invoice_adminkey(client, invoice, adminkey_headers_from):
     response = await client.post(
         "/api/v1/payments", json=data, headers=adminkey_headers_from
     )
-    assert response.status_code < 300  # should pass
+    if is_regtest:
+        # no self payments
+        assert response.status_code == 520
```

Comment:
This is a sad test case but true :(

We need to create the invoice somewhere else (in another lnbits instance :) and pay it to be sure this test case does it's job.

Reply:
i agree, can we but those 2 things into an issue, and we will figure it out later? i really need to move forward with boltz extension and i need to PR to make tests.